### DISCUSSION TITLE
开取一键下载视频(有人盗我视频,我要存证据)

### DISCUSSION THREAD

**Discussion by akashineedgem** [leiurayer/downkyi#368] [Ideas]
Hello author,actually I am a victim of a content creator.I want to backup all the proof and video that bilibili steal my video. IF u afraid then add more watermark inside the video nevermind instead I GLADLY WELCOME THIS OPTION AS I WANT TO PROOF THAT THIS DUDE STEAL MY VIDEO AND ADD WTERMARK MONETIZE IT!

**Comment by leiurayer** [comment]
Sorry about that. The original intention of this app is to collect favorite videos, not to steal videos. Using other tools to steal videos is also illegal. I can help if you need.

**Comment by akashineedgem** [comment]
I need help man， I am tired and exhausted， It is not fair wasted 3 months
finding solutions , got tied by works and volunteer overtime while this guy
wants to ruin my hobby. i posted all the evidences and other youtuber
pieces of evidence but no avail.

**Comment by akashineedgem** [comment]
I dont care I know I AM SHAMELESS! BUT AT LEAST I WANT TO KEEP ALL THIS
CHANNEL VIDEOS! THAT THIS SHAMELESS DUDE IS STEALING FROM ME AND OTHER
PEOPLE!

**Comment by akashineedgem** [comment]
NO！ I am not stealing videos, Actually it is the BILIBILI ARE STEALING MY
VIDEOS! I am finding tools that help me download all the videos to act as a
proof that this guy keep monitoring us and deleting videos when exposed. He
is not just stealing my video but a lot of youtube creator!!!! I mean a lot
since I screenshot that HE HAS 1000+VIDEOS 0.1+ BILLION VIEWS!
https://space.bilibili.com/6377073/
this guy steal my videos ！monetize it！ When I complain nothing happens,just
blacklisted me! He also copyright strike me! The irony is that this guy
upload date is later than me! YOUTUBE ACCEPT THE COPYRIGHT STRIKE BECAUSE
HE HAS 200K+SUBSCRIBER ! WHILE I HAVE ONLY 1.7k at that time! I am already
wasted a lot of time last year keep sending email to youtube nothing
happens!









https://space.bilibili.com/6377073/
https://space.bilibili.com/6377073/ THIS GUY KEEP CHANGING HIS FKING NAME!
AND BILIBILI STILL PROTECT THIS GUY SAYING THAT NAH THIS GUY IS NOT
STEALING! HE GOT AUTHORIZED PERMISSION! YA AUTHORISED PERMISSION UNTIL A
LOT OF YOUTUBER POST HIS CHANNEL NAME SAY THAT THIS GUY ILLEGAL UPLOAD
WITHOUT PERMISSION!

我知道一点点华语！不知道你可不可以帮帮我。你是中国人吗？

**Reply by HighVorz** [reply]
you can register bilibili and upload your videos as in youtube, it do helps to your copyright.
